README.md

```diff
@@ -1,86 +1,22 @@
         <a href="https://codecov.io/gh/TypeScriptToLua/TypeScriptToLua"><img alt="Coverage" src="https://img.shields.io/codecov/c/gh/TypeScriptToLua/TypeScriptToLua.svg?logo=codecov" /></a>
         <a href="https://discord.gg/BWAq58Y"><img alt="Chat with us!" src="https://img.shields.io/discord/515854149821267971.svg?colorB=7581dc&logo=discord&logoColor=white"></a>
     </h1>
+    <a href="https://typescripttolua.github.io/" target="_blank">Documentation</a>
+    |
+    <a href="https://typescripttolua.github.io/play/" target="_blank">Try Online</a>
+    |
+    <a href="https://github.com/TypeScriptToLua/TypeScriptToLua/blob/master/CHANGELOG.md">Changelog</a>
+    |
+    <a href="https://github.com/TypeScriptToLua/TypeScriptToLua/blob/master/CONTRIBUTING.md">Contribution guidelines</a>
 </div>
 
-A generic TypeScript to Lua transpiler. Write your code in TypeScript and publish Lua!
-
-Large projects written in lua can become hard to maintain and make it easy to make mistakes. Writing code in TypeScript instead improves maintainability, readability and robustness, with the added bonus of good IDE support. This project is useful in any environment where Lua code is accepted, with the powerful option of simply declaring any existing API using TypeScript declaration files.
-
-## Documentation
-
-More detailed documentation and info on writing declarations can be found [on the wiki](https://github.com/TypeScriptToLua/TypescriptToLua/wiki).
-
-Changelog can be found in [CHANGELOG.md](https://github.com/TypeScriptToLua/TypescriptToLua/blob/master/CHANGELOG.md)
-
-## Usage Guide
-
-**Install**
-
-`npm install -g typescript-to-lua`
-
-**Compile Files**
-
-`tstl path/to/file.ts path/to/other-file.ts`
-
-**Compile Projects**
-
-`tstl -p path/to/tsconfig.json`
-
-**Compile project in watch mode**
-
-`tstl -p path/to/tsconfig.json --watch`
+---
 
-**Example tsconfig.json**
-
-```json
-{
-  "compilerOptions": {
-    "target": "esnext",
-    "lib": ["esnext"],
-    "strict": true
-  },
-  "tstl": {
-    "luaTarget": "JIT"
-  }
-}
-```
-
-## Contributing
-
-All contributions are welcome, but please read our [contribution guidelines](https://github.com/TypeScriptToLua/TypescriptToLua/blob/master/CONTRIBUTING.md)!
-
-## Declarations
-
-The real power of this transpiler is usage together with good declarations for the Lua API provided. Some examples of Lua interface declarations can be found here:
-
-- [Dota 2 Modding](https://github.com/ModDota/API/tree/master/declarations/server)
-- [Defold Game Engine Scripting](https://github.com/dasannikov/DefoldTypeScript/blob/master/defold.d.ts)
-- [LÖVE 2D Game Development](https://github.com/hazzard993/love-typescript-definitions)
-- [World of Warcraft - Addon Development](https://github.com/wartoshika/wow-declarations)
-- [World of Warcraft Classic - Addon Development](https://github.com/wartoshika/wow-classic-declarations)
-
-## Sublime Text integration
```

Comment:
It might be nice to instead have a list of editors (for TypeScript I guess?) that can be used.